packages/components/accordion/src/AccordionHeader/AccordionHeader.tsx

```diff
@@ -1,47 +1,54 @@
 import React, { FC } from 'react';
-import cn from 'classnames';
+import { cx } from 'emotion';
 import { Subheading } from '@contentful/f36-typography';
 import type { HeadingElement } from '@contentful/f36-typography';
 import { ChevronDownTrimmed } from '@contentful/f36-icons';
+import type { CommonProps } from '@contentful/f36-core';
 
-import styles from '../Accordion.css';
+import getStyles from '../Accordion.styles';
 
-export interface AccordionHeaderProps {
+export interface AccordionHeaderProps extends CommonProps {
   /**
    * Child nodes to be rendered in the component
    */
   children?: React.ReactNode;
-  /**
-   * An ID used for testing purposes applied as a data attribute (data-test-id)
-   */
-  testId?: string;
   /**
    * The function that will be called once the user clicks on the accordion title
    */
-  handleClick: VoidFunction;
+  handleOnClick: VoidFunction;
   /**
    * A boolean that tells if the accordion should be expanded or collapsed
    */
   isExpanded: boolean;
   /**
    * An unique id that is necessary for the aria roles and properties
    */
-  ariaId: number | null;
+  ariaId: string;
   /**
    * The heading element that will be used by the Subheading component
    */
   element?: HeadingElement;
+  /**
+   * Specify the alignment of the chevron inside the accordion header
+   */
+  align?: 'start' | 'end';
 }
 
 export const AccordionHeader: FC<AccordionHeaderProps> = ({
   children,
-  handleClick,
+  handleOnClick,
   isExpanded = false,
   ariaId,
   element = 'h2',
+  align = 'end',
 }: AccordionHeaderProps) => {
-  const classNames = cn(styles.AccordionHeader, {
-    [styles['AccordionHeader--expanded']]: isExpanded,
+  const styles = getStyles();
```

Comment:
We need to have several functions.. one to return accordion styles, one to return accordion header, and one to return accordion items.